Sources/App/Models/KarmaMessage.swift

```diff
@@ -14,6 +14,31 @@ struct KarmaMessage {
 
 extension KarmaMessage {
 
+    private func messageColor() -> String {
+        return karma >= 0 ? "#36a64f" : "#E8B122"
+    }
+
+    private func crease() -> String {
```

Comment:
maybe better to use "changed" which reads more naturally at point of use.

Reply:
Or maybe change type ? I agree crease is not readable